docs/reference/contributing/connectivity/MeshInterface.md

```diff
@@ -170,6 +170,67 @@ State|Description
 
 ![tx](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/tx_process.png)
 
+### MESH PHY extended driver support
+
+Mesh IEEE 802.15.4 MAC will automatically use RF driver extended features. These features are needed when MAC user need to send IEEE 802.15.4 frames. MAC extensuon will enable if driver not support extended features. 
+
+Extented driver features enable next feature's:
+
+- MAC to control packet TX time define CCA backoff period before CCA check
```

Comment:
should the features be 
MAC controlled CCA/CSMA ?
support for IEEE 802.15.4-2015 enhanced ACK and information elements?

Reply:
OK